content/html-css/blocks/what-is-css/index.md

```diff
@@ -0,0 +1,118 @@
+
+</details>
+
+<details>
+<summary> What is a selector, property, and value?</summary>
+
+```css
+selector {
+  property: value;
+}
+```
+
+Selectors look like this: `p { background: red }`. The selector is the `p` in this case. This p selects all elements of the DOM tree called `p` and sets a rule about how to show them. Selectors always come first in a ruleset.
+
+A property is a quality, a characteristic. A value is the amount.
+
+For a person, you might say their 'age' property has the value of 40. For an element, you might set the `width` property to the value of `100%`.
+
+</details>
+
+<details>
+<summary>How do you know what properties and values there are?</summary>
+
+You learn them by using them. You look it up here: https://developer.mozilla.org/en-US/docs/Web/CSS/Reference. And the other thing you can do is look at the autocomplete in Devtools.
+
+</details>
+
+<details>
+<summary>Why is CSS a 'cascading' style sheet? What does cascade mean?</summary>
+
+<iframe src="https://giphy.com/embed/zFSGphnnT9opa" width="480" height="480" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/zFSGphnnT9opa">via GIPHY</a></p>
+
+A cascade is a stream or sequence of a things where each thing affects the next. You might imagine pouring champagne into a tower of glasses. The champagne flows into the containers; it cascades. In CSS we’re actually pouring from many bottles into one tower.
+
+So in CSS 'the cascade' means the order or sequence of the rules as they flow into the DOM tree according to CSS precedence.
+
+</details>
+
+<details>
+<summary>What does precedence mean?</summary>
+
+It means hierarchy. One hierarchy in CSS is inheritance: children inherit from parents. If a parent node has red text, all its children will inherit that red text color. But when another rule comes after, and overrules, it takes precedence. Everything in CSS is determined by order.
+
+Order is so important in CSS because CSS is a declarative programming language that programs the layout of boxes.
+
+CSS renders the nodes in our DOM tree as nested boxes, and we program their appearance using fonts, colours, and spacing.
+
+</details>
+
+<details>
+<summary>What does declarative mean?</summary>
+
+It means, broadly speaking, that the entire code is processed and then applied, from top to bottom in one go. You don’t write for loops or if/else statements; there are no functions as such; you just _declare_ the outcome.
+
+In CSS you do the 'what', and the browser does the 'how'.
+
+</details>
+
+<details>
+<summary>What is specificity?</summary>
+
+Specificity is a [weighting system for CSS rules](https://specifishity.com/). The more specific your selector, the more power it has. For example: IDs can only ever mean _one_ node. IDs are unique, so they are _more specific_ than classes, which could mean many nodes.
```

Comment:
```suggestion
Specificity is a [weighting system for CSS rules](https://specifishity.com/).

The more specific your selector, the more power it has. For example: an `id` attribute can only ever be attached to _one_ node. This means IDs are unique, so they are _more specific_ than classes, which could mean many nodes.
```

I also wonder if it would be worth swapping out "node" for "element" here? Above we use "element", and my impression is that is the more broadly understood term here.